src/main/java/org/isf/visits/gui/VisitView.java

```diff
@@ -296,17 +329,42 @@ private JButton getAddVisitFirstButton() {
 		}
 		return addFirstVisitButton;
 	}
+	
+	private JButton getDeleteFirstVisitButton() {
+		if (deleteFirstVisitButton == null) {
+			deleteFirstVisitButton = new JButton(MessageBundle.getMessage("angal.visit.removevisit.btn"));
+			deleteFirstVisitButton.setMaximumSize(new Dimension(VISIT_BUTTON_WIDTH, ALL_BUTTON_HEIGHT));
+			deleteFirstVisitButton.setHorizontalAlignment(SwingConstants.LEFT);
+			deleteFirstVisitButton.addActionListener(actionEvent -> {
+
+				int row = jTableFirst.getSelectedRow();
+				if (row < 0) {
+					MessageDialog.info(VisitView.this, MessageBundle.getMessage("angal.common.pleaseselectarow.msg"));
```

Comment:
The `MessageDialog.info()` method includes the `MessageBundle.getMessage()` so only pass `angal.common.pleaseselectarow.msg` and not the result of the lookup.

Reply:
Done, thanks!